core/arch/arm/pta/sdp_pa.c

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (C) 2017, ARM Limited, All Rights Reserved
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <kernel/pseudo_ta.h>
+#include <mm/core_memprot.h>
+#include <mm/tee_mmu.h>
+#include <sdp_pa.h>
+#include <string.h>
+
+#define PTA_NAME "sdp_pa.pta"
+
+static TEE_Result sdp_pa_cmd_virt_to_phys(uint32_t types,
+		TEE_Param params[TEE_NUM_PARAMS])
```

Comment:
minor: could you indent the parameter lines nearer to 1st param position. Make the code more clear (to me :) and a bit more consistent with (almost) other core source files. (at least 2 more tabs should be fine).
Here and `open_session`/`invoke_command`.

Reply:
Yes, I will fix it.